source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+==========================================
+How To Write Code And Copy Review Comments
+==========================================
+
+Summary
+-------
+
+- Be kind.
+- Explain your reasoning.
+- Balance giving explicit directions with just pointing out problems and letting
+  the author decide.
+- Encourage authors to simplify code or add code comments instead
+  of just explaining the complexity to you.
+- Encourage authors to simplify explanations when appropriate. We're all
+  smart, but smart words take more time to read. Readers prefer
+  conciseness and the simplest explanation in most situations.
+
+Label Your Comments
++++++++++++++++++++
+
+Label comments similar to `Conventional Comments <https://conventionalcomments.org/>`__
+so the author understands this is a suggestion, issue, question, or comment.
+
+- **No "nits"**.
+- Issue: Something is logically, factually, or grammatically incorrect and needs
+  to be corrected. Issues are **always** blocking and no PR should be merged with
+  an open issue.
+- Question: You (the reviewer) have a question about something in a PR.
+  **Do not use** questions as in a Socratic style. If there is a logical, factual, or
```

Comment:
I'm not sure I know what to add if there's something that doesn't make sense to me, whether it's knowledge of the product or how it's worded. I think questions in Socratic style and genuine questions are difficult to distinguish through a written medium.